models/shopkeeper.js

```diff
@@ -0,0 +1,37 @@
+const Joi = require('joi');
+const mongoose = require('mongoose');
+ 
+const Shopkeeper = mongoose.model('Shopkeeper', new mongoose.Schema({
+    name: {
+        type: String,
+        required: true,
+        minlength: 5,
+        maxlength: 50
+    },
+    phone: {
+        type: String,
+        required: true,
+        minlength: 10,
+        maxlength: 15,
+        unique: true
+    },
+    shopname: {
```

Comment:
@uppal7 
Please use proper naming convention, refer to this link https://stackoverflow.com/questions/5916080/what-are-naming-conventions-for-mongodb